bindata/network/frr-k8s/frr-k8s.yaml

```diff
@@ -86,7 +86,7 @@ spec:
         - --node-name=$(NODE_NAME)
         - --namespace=$(NAMESPACE)
         - --metrics-bind-address=127.0.0.1:7572
-        - --log-level=info
+        - $(LOG_LEVEL)
```

Comment:
Yes, my original version of this was with a "bash -c" command.

My reasoning here was that this is more "automation ready", or even if we keep manual, less prone to errors when we align the manifests (provided that we want to keep the upstream manifests as source of truth).

This comes with the price of not being able to inject any string we want, which does obviously not mean that we won't need any additional parameters in the future :P .

In that case imo (taking the node selector of where we want to run those pods for example), it may make sense to expose them as fields of the CRD instead of asking the user to create a configmap and kill the pods (which is legit for flipping the log level as it's probably a scenario where investigation is required).

Lastly, not sure I understood the "per node flexibility" part.

Reply:
Yeah, this mechanism is only meant for troubleshooting parameters. So whatever we decide to do with it is for our own convenience and that's the perspective I had with my comments. No way we would use this for the node selector.

The per node flexibility is acting on specific nodes rather than all. So env-overrides would be a config map which would have keys for each node. For example, if you are going to enable chatty logs on an environment where you would not like to impact cluster-wide performance, you perhaps only one to enable the logs on one node. We actually have an example in the README for ovn-k:
https://github.com/openshift/cluster-network-operator/blob/eea8932fa0503a647d6072ea1c016fb7a853aba0/README.md?plain=1#L176-L197

Reply:
ah now I get it! I don't think we need this level of log granularity here. The verbosity (and the impact on the performances) are going to be the same as what we have to metallb (mostly frr logs) and this never was a problem (while I bet ovn can be way more chatty).

Reply:
so, all in all, if it's not a blocker I'd stick with this version for the reasons I wrote above